src/scout_apm/core/call_set.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+import logging
+from datetime import datetime
+
+import scout_apm.core.backtrace
+
+# Logging
+logger = logging.getLogger(__name__)
+
+
+class NPlusOneCallSet():
+    def __init__(self):
+        self.items = {}
+
+    def update(self, sql_string, call_count, call_duration):
+        self.find_for(sql_string).update(call_count, call_duration)
+
+    def capture_backtrace(self, sql_string):
```

Comment:
This method name is misleading because it does not capture a backtrace - it returns true/false for if a trace should be captured?

Also probably a missing `return` on next line.